misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -3886,6 +3904,17 @@ private <T extends Node> boolean isMultilineModuleMember(T node) {
         }
     }
 
+    private <T extends Node> boolean isMemberOfScope(T node) {
+        switch (node.kind()) {
+            case OBJECT_METHOD_DEFINITION:
+            case RESOURCE_ACCESSOR_DEFINITION:
+            case OBJECT_FIELD:
+                return true;
+            default:
+                return false;
+        }
+    }
```

Comment:
The private function's name: `isMemberOfScope` and its implementation are quite mismatched to me. What I see here is, that we are checking the node kind across a set of nodes and returning a true/false. What is the intention of this check?

Reply:
This issue is related to functions defined within a scope(service classes, classes, services). Thus here given a Node it identifies whether is node is in a scope.

Reply:
Got it!